tests/testlibrary.c

```diff
@@ -274,7 +290,7 @@ test_languages_config (void)
   g_assert_no_error (error);
   g_assert_cmpstr (value[0], ==, "en");
 
-  res = flatpak_installation_set_config_sync (inst, "extra-languages", "en;pt", NULL, &error);
+  res = flatpak_installation_set_config_sync (inst, "extra-languages", "pt_BR", NULL, &error);
```

Comment:
I think you need to add a `clean_extra_languages()` at the end of this function if the goal is to leave the config in the state you found it. That way other unit tests aren't affected by leftover state from this one.

Reply:
Also, I think it would be good to test one of the longer locales and a language too, like "en;pt_BR;az_Latn_AZ"

Reply:
yes, that would be excellent